test-commonjs.cjs

```diff
@@ -18,13 +18,13 @@ const symbol = 'BTC/USDT:USDT';
 async function main() {
     try {
         // test cjs version
-        const exchange = new ccxt.bybit({});
+        const exchange = new ccxt.gate({});
```

Comment:
bybit failed too frequently: https://app.travis-ci.com/github/ccxt/ccxt/builds/262664322#L2650

Reply:
@ttodua I think this will fix it https://github.com/ccxt/ccxt/pull/17701

Reply:
ah, havent seen it. ok reverted

Reply:
btw, it is worth to have bybit there (which needs to have a proxy)? maybe we could replace it with non proxy needed one. like gate or other.

Reply:
i think it is ok, and at least if the proxy stops working we can easily spot it